packages/synapse-react-client/src/components/TermsAndConditions/TermsAndConditions.tsx

```diff
@@ -1,116 +1,74 @@
 import React, { useEffect, useState } from 'react'
-import { tcItem } from './TermsAndConditionsItem'
+import { LoadingItem, tcItem } from './TermsAndConditionsItem'
 import TermsAndConditionsItem from './TermsAndConditionsItem'
 import { Button, Link } from '@mui/material'
-import { ChatBubblesIcon } from '../../assets/icons/terms/ChatBubblesIcon'
-import { LockIcon } from '../../assets/icons/terms/LockIcon'
-import { ScaleIcon } from '../../assets/icons/terms/ScaleIcon'
-import { ShieldIcon } from '../../assets/icons/terms/ShieldIcon'
-import { PeopleIcon } from '../../assets/icons/terms/PeopleIcon'
-import { PenIcon } from '../../assets/icons/terms/PenIcon'
-import { SpeakerIcon } from '../../assets/icons/terms/SpeakerIcon'
-import { FlagIcon } from '../../assets/icons/terms/FlagIcon'
-import { URL_TERMS_CONDITIONS_AGREEMENT } from '../../utils/SynapseConstants'
+import {
+  BUNDLE_MASK_QUERY_RESULTS,
+  URL_TERMS_CONDITIONS_AGREEMENT,
+} from '../../utils/SynapseConstants'
+import { useGetFullTableQueryResults } from '../../synapse-queries'
+
+export const termsAndConditionsTableID = 'syn51718002'
 
 export type TermsAndConditionsProps = {
   onFormChange: (formComplete: boolean) => void
   hideLinkToFullTC?: boolean
 }
 
-const dataUseLink =
-  'https://s3.amazonaws.com/static.synapse.org/governance/SynapseCommonsDataUseProcedure.pdf?v=4'
-
-const tcList: tcItem[] = [
```

Comment:
Kept the existing data tcItem data model, with the exception of the icon (which now points to a file handle ID)